Firmware/MVP/battsci.cpp

```diff
@@ -15,7 +15,19 @@
 uint8_t BATTSCI_state = STOPPED;
 
 uint8_t spoofedVoltageToSend = 0;
-int16_t spoofedCurrentToSend = 0; //JTS2doLater spoofed pack current can probably be int8_t (+127 A)
+int16_t spoofedCurrentToSend = 0;     //JTS2doLater spoofed pack current can probably be int8_t (+127 A)
+
+uint8_t LiBCM_SoC = 60;
+
+byte SoC_Bytes[] = {0x15, 0x50};      // SoC Bytes to send to MCM.  Index 0 is the upper byte and index 1 is the lower byte.
+byte temperature_Byte = 0x33;         // Temperature Byte to send to MCM.  0x33 is +21 Degrees C.
+uint16_t calculatedSoC = 500;
+uint16_t oldCalculatedSoC = 500;
+
+bool initializeSoC = true;
```

Comment:
After reading the other comment I think I need to rename this bool:
initializeSpoofedSoC

It's only used for an if statement that runs once and resets the spoofed SoC values so that the spoofed SoC LiBCM is sending to the MCM gets updated after key-on, especially after grid charging or if the car sat for 2 weeks and SoC went down a lot.

Reply:
Instead of having this variable (initializeSpoofedSoC), couldn't we just call SoC_getBatteryStateNow_percent()?  The main thing I don't understand is why there needs to be a separate 'spoofed' SoC, except to remap the actual SoC.  You stated elsewhere that the goal is to prevent background recharge when the actualSoC reaches 60%... and to do that we need to spoof 70%.  But instead of maintaining an entirely separate variable, we should just make the remap function remap 60% to 70%... unless there are other reasons we need to spoof SoC?